src/coreclr/vm/jitinterface.cpp

```diff
@@ -2855,6 +2889,53 @@ void CEEInfo::ScanTokenForDynamicScope(CORINFO_RESOLVED_TOKEN * pResolvedToken,
     ScanToken(pModule, pResolvedToken, th, pMD);
 }
 
+OBJECTHANDLE CEEInfo::getJitHandleForObject(OBJECTREF obj)
```

Comment:
```suggestion
CORINFO_OBJECT_HANDLE CEEInfo::getJitHandleForObject(OBJECTREF obj)
```
This returns JIT handle, not the VM `OBJECT_HANDLE`.